From eae77aa83b7f8301f03f09a3fc6695daaf9134b8 Mon Sep 17 00:00:00 2001
From: Sergii Piatakov <sergii.piatakov@globallogic.com>
Date: Thu, 15 Nov 2018 15:28:04 +0200
Subject: [PATCH 1/2] fix truncation error

To convert float to integer the truncation is performed, but the
rounding is expected.

Test: Add (4.9, 4.9) should return 10.
Signed-off-by: Sergii Piatakov <sergii.piatakov@globallogic.com>

	modified:   calculator.cpp
---
 calculator.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/calculator.cpp b/calculator.cpp
index b91afea..208c9c3 100644
--- a/calculator.cpp
+++ b/calculator.cpp
@@ -3,6 +3,7 @@
 int Calculator::Add (double a, double b)
 {
     return a + b;
+
 }
 
 int Calculator::Sub (double a, double b)
-- 
2.17.1

